packages/app/src/sandbox/eval/transpilers/svelte/svelte-worker.js

```diff
@@ -47,9 +48,23 @@ function getV2Code(code, path) {
   return { code: compiledCode, map };
 }
 
+function getV3Code(code) {
+  self.importScripts(['https://unpkg.com/svelte@3.0.1/compiler.js']);
+  try {
+    return window.svelte.compile(code, {
+      dev: true,
+    }).js;
+  } catch (e) {
+    return self.postMessage({
```

Comment:
This is exactly how it's supposed to work! 😄

Reply:
It feels dirty ahah